development/product/jumpstart-engineering/contrib.make

```diff
@@ -179,7 +179,7 @@ projects[token][version] = "1.6"
 projects[transliteration][subdir] = "contrib"
 projects[transliteration][version] = "3.2"
 projects[uuid][subdir] = "contrib"
-projects[uuid][version] = "1.0-beta1"
+projects[uuid][version] = "1.0-beta2"
```

Comment:
This needs to stay 1.0-beta1 as 1.0-beta2 has a dependency on php 5.5:
http://cgit.drupalcode.org/uuid/commit/?id=f7b7549